### ISSUE TITLE
Clone page

### ISSUE THREAD

**Issue by jenstornell** [k-next/plugins#22]
A popular request and 3 plugins created for Kirby 2.

One plugin should be enough.

https://github.com/jenstornell/kirby-plugins/projects/1#column-1671490

**Comment by wottpal** [comment]
This is also something where I think a core-component would be a better fit. Regarding the UI I could imagine two scenarios:

- Whenever the user creates a page there is a *Clone From* Page-Select field at the bottom of the Modal. 
- An additional option *Duplicate* for each page where a modal pops up and asks about the new location. (Only locations where the template-type is accepted can be chosen.)

Is one of these accomplishable with the current plugin-architecture? What do you think @bastianallgeier? 3.x or plugin?

**Comment by bnomei** [comment]
we had this discussion with kirby 2 in the forums as well. problem is duplication needs to trigger a lot of stuff like checking slug, sorting etc. i also think this might be better 'core' and thus well supported.

**Comment by bastianallgeier** [comment]
I agree that this should be a core feature, but I see the same issues as @bnomei. It's not trivial. Like moving pages to different parents. I like the idea to add this to the "Create new page" dialog to be honest. More like a template for a new page than a duplicate. This would make a lot of things easier.

**Comment by bnomei** [comment]
maybe it should be called „duplicate as draft“? that would make its function clearer.

**Comment by jenstornell** [comment]
@bnomei I'm not sure about renaming the issue. It's still the idea. But instead I've added the information to the initial post/issue.

>Possible solutions:
Create new page with a "template" which in this case is content from another page

**Comment by bnomei** [comment]
@jenstornell i meant the label of the panel menu item not renaming this issue. 
sorry about being unclear. i edited my post above to reflect this.
